src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -25,6 +25,10 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
             var stringType = GetSpecialType(SpecialType.System_String, diagnostics, node);
             var objectType = GetSpecialType(SpecialType.System_Object, diagnostics, node);
             var intType = GetSpecialType(SpecialType.System_Int32, diagnostics, node);
+
+            // PROTOTYPE follow up with Neal on the implementation 
+            ConstantValue? resultConstant = null;
+            bool constantEnabled = true;
```

Comment:
Think the code would read better if the names of these paired values matched better. For instance `resultConstant` and `isResultConstant`

Reply:
Thanks for the suggestion, changes made.

---
In reply to: [447267474](https://github.com/dotnet/roslyn/pull/45396#discussion_r447267474) [](ancestors = 447267474)